### ISSUE TITLE
On demand stores

### ISSUE THREAD

**Issue by RiadhAdrani** [sindresorhus/electron-store#280]
Hello,

I am just wondering, if I can create multiple stores on demand, like some sort of tables in SQL, which will help reduce pressure on one file.

Are there any drawbacks?
Thanks.

**Comment by sindresorhus** [comment]
Yes. Pass a different `name` per store, for example `new Store({name: 'users'})`, and each gets its own JSON file in the same cwd.

The drawbacks are that every `set()` rewrites the whole file, and that with `watch: true` each store watches its own file.
